### PR TITLE
Created new pipeline for webui PRs

### PR DESCRIPTION
**Pull request by KirbyKatcher**

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: pipelines/pipelines/galasa-ui/build-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    workspaces:
+     - name: output
+       workspace: git-workspace
+  #----------------------------------------------------------------
+  # run unit tests
+  - name: test-webui
+    taskRef:
+      name: unix-command
+    runAfter:
+    - generate-api
```
Comment (eamansour) [inline pipelines/pipelines/galasa-ui/build-pr.yaml:74]: Probably needs changing from `generate-api` to `clone-webui`

**Review by eamansour — CHANGES_REQUESTED** [review]
Looks good - just spotted a couple of changes that should get the pipeline working

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**